RecoLocalTracker/Phase2TrackerRecHits/plugins/Phase2TrackerRecHits.h

```diff
@@ -0,0 +1,34 @@
+#ifndef RecoLocalTracker_Phase2TrackerRecHits_Phase2TrackerRecHits_h
+#define RecoLocalTracker_Phase2TrackerRecHits_Phase2TrackerRecHits_h
+
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/DetId/interface/DetId.h"
+
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+
+#include "RecoLocalTracker/Records/interface/TkStripCPERecord.h"
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEDummy.h"
+
+
+  class Phase2TrackerRecHits : public edm::global::EDProducer<> {
```

Comment:
it is actually not worth to make a header file for this small piece, it can be fully embedded in Phase2TrackerRecHits.cc
.. similar story for a few other plugins/\*.h files, which are meant to be included by only one .cc